packages/component-submission/server/aggregates/Submission.js

```diff
@@ -17,6 +21,9 @@ class Submission {
     )
     this.files = await this.FileModel.findByManuscriptId(manuscriptId)
     this.teams = await this.TeamModel.findByManuscriptId(manuscriptId)
+    this.suggestions = await this.SemanticExtraction.findByManuscriptId(
```

Comment:
do we need to change the graphQL?

Reply:
Isnt the point of graphql that we dont have to change it, we only need to handle the response at the client?